tests/btrfs-progs/install.pm

```diff
@@ -24,22 +24,17 @@ sub log_create {
 }
 
 sub install_dependencies {
-    my @deps = qw(
-      autoconf
-      automake
-      lzo-devel
-      git-core
-      gcc
-      libblkid-devel
-      zlib-devel
-      libext2fs-devel
-      libmount-devel
-      libuuid-devel
-    );
-    if (get_var('BTRFS_PROGS_DEPS')) {
-        @deps = split(/,/, get_var('BTRFS_PROGS_DEPS'));
+    my $transaction = shift;
+    if ($transaction) {
+        my @deps = split(/ /, get_var('BTRFS_PROGS_DEPS', 'attr'));
+        foreach my $dep (@deps) { trup_call("--continue pkg install $dep"); }
+    }
+    else {
+        my $deps = 'autoconf automake lzo-devel git-core gcc libblkid-devel ' .
+          'zlib-devel libext2fs-devel libmount-devel libuuid-devel';
```

Comment:
I think the benefit of each line a package name could trace the git commit info about the reason that we added that package, I suggest keeping it.